lib/ssh_scan/database/mongo_db.rb

```diff
@@ -0,0 +1,39 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module   SSHScan
+  module Database
+    class MongoDb
+      def initialize
+        @db = Mongo::Client.new(['127.0.0.1:27017'], :database => 'api_db')
+        @scans = @db[:scans]
+      end
+
+      def add_scan(worker_id, uuid, result)
+        @scans.insert_one("worker_id" => worker_id, "uuid" => uuid,
+                          "scan" => result.first, "scanned_on" => Time.now)
+      end
+
+      def delete_scan(worker_id, uuid)
+        @scans.delete_one(:worker_id => worker_id, :uuid => uuid)
+      end
```

Comment:
Same questions here, as I asked in SQLite abstraction about whether worker ID is relevant in this context.